src/Visualization.jl

```diff
@@ -1,7 +1,9 @@
 using Graphs: SimpleGraph, Edge
 using GraphMakie: graphplot, GraphPlot
 using Combinatorics: combinations
+
 import Makie
+export draw
```

Comment:
```suggestion
```

The `draw` function was the legacy name of what is now implemented in `Makie.plot!(P::GraphPlot{Tuple{GenericTensorNetwork}})`